src/main/cpp/t03_points.cpp

```diff
@@ -23,10 +23,32 @@
 
 #include "t03_points.h"
 #include <iostream>
-
+#include <set>
+#include <map>
 
 using namespace std;
 
 int t03_points() {
-    
+    int n;
+    float l;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'l' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4684787)